guide/private-key-management/cloud-backup.md

```diff
@@ -33,14 +33,16 @@ https://www.figma.com/community/file/995256542920917246/BDG---Private-key-manage
    layout = "full-width"
 %}
 
-# Automatic cloud backup
+# Cloud backup
 
 By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new [bitcoin wallet]({{ '/guide/glossary/wallet/#bitcoin-wallet' | relative_url }}). This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) that is used to initiate a wallet.
 
-This scheme can be a good fit for wallets meant for storing smaller amounts of bitcoin. This includes wallets for novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds. Wallets using automatic cloud backups also work well for what experienced users might call *hot* or *spending* wallets where convenience is more important than maximum security. 
+This scheme can be a good fit for wallets meant for storing smaller amounts of bitcoin. This includes wallets for novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds. Wallets using automatic cloud backups also work well for what experienced users might call *hot* or *spending* wallets where convenience is more important than maximum security.
 
 The automatic cloud backup scheme gives users reasonable security against theft, while providing low risk for self-inflicted losses.
 
+If a wallet supports Lightning, it may already have cloud backup [enabled for the channel state]({{'/guide/onboarding/backing-up-a-wallet/cloud-backup/' | relative_url}}); therefore, it would not take much additional work to store the private key in the cloud as well.
```

Comment:
"it would not take much work" - is there any chance we can explain this a bit? What kind of work process is involved in this?

Reply:
That's an assumption I am making (perhaps wrong). What I'm trying to express is that if the developer has already built a way for the wallet app to communicate with a cloud provider for frequent channel state updates, then it seems to me like it would not require much additional coding to also store a backup of the key as well.

Maybe something like this:

> If you have implemented cloud backups for Lightning channel state in your app, you could also use this same conection with the cloud provider to store a backup of the user's private key.

Reply:
Done